contrib/storage-cassandra/src/test/java/org/apache/drill/exec/store/cassandra/TestCassandraSuit.java

```diff
@@ -65,7 +66,10 @@ public static void tearDownCluster() {
 
   private static void startCassandra() {
     cassandra = new CassandraContainer<>("cassandra")
-        .withInitScript("queries.cql");
+      .withInitScript("queries.cql")
+        .withStartupTimeout(Duration.ofMinutes(2))
+          .withEnv("CASSANDRA_SNITCH", "GossipingPropertyFileSnitch") // Tune Cassandra options for faster startup
+            .withEnv("JVM_OPTS", "-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dcassandra.initial_token=0");
```

Comment:
Possibly we can do event better here:
```
-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dcassandra.load_ring_state=false -Dcassandra.initial_token=1 -Dcassandra.num_tokens=nil -Dcassandra.allocate_tokens_for_local_replication_factor=nil
```
https://github.com/sky-uk/cassandra-operator/issues/103#issuecomment-499040927